core/src/wallet/ethereum/explorers/EthereumLikeBlockchainExplorer.h

```diff
@@ -121,7 +121,10 @@ namespace ledger {
             virtual Future<std::shared_ptr<BigInt>> getBalance(const std::vector<EthereumLikeKeychain::Address> &addresses) = 0;
             virtual Future<std::shared_ptr<BigInt>> getGasPrice() = 0;
             virtual Future<std::shared_ptr<BigInt>> getEstimatedGasLimit(const std::string &address) = 0;
-            virtual Future<std::shared_ptr<BigInt>> getERC20Balance(const std::string &address, const std::string &erc20Address) = 0;
+            virtual Future<std::shared_ptr<BigInt>> getERC20Balance(const std::string &address,
+                                                                    const std::string &erc20Address) = 0;
+            virtual Future<std::vector<BigInt>> getERC20Balances(const std::string &address,
```

Comment:
I suggest to rename `getERC20Balances` to `getAllERC20Balance` because the current name is really to close to `getERC20Balance`